### ISSUE TITLE
FootballAction ppTSM pretrain model multiple outputs are not found in the ppTSMHead class

### ISSUE THREAD

**Issue by muare** [PaddlePaddle/PaddleVideo#360]
**Describe the bug**

the output "name: save_infer_model/scale_1.tmp_1" is used for feature extraction before BMN traing. 
But this output is not defined in pptsm_head.py.

Would you please help to clarify?

**To Reproduce**
1. the pretrain model downloaded from download.txt under "PaddleVideo/applications/FootballAction/checkpoints"
# pptsm
https://videotag.bj.bcebos.com/PaddleVideo-release2.1/FootballAction/pptsm.tar

<img width="354" alt="image" src="https://user-images.githubusercontent.com/1112172/153758125-09cf2308-8190-48c2-95be-5bee4e46992b.png">

2. but the ppTSMHead class seems have nothing about the feature map output, 

    def forward(self, x, num_seg):
        """Define how the head is going to run.
        Args:
            x (paddle.Tensor): The input data.
            num_segs (int): Number of segments.
        Returns:
            score: (paddle.Tensor) The classification scores for input samples.
        """

        #XXX: check dropout location!
        # [N * num_segs, in_channels, 7, 7]
        x = self.avgpool2d(x)
        # [N * num_segs, in_channels, 1, 1]
        if self.dropout is not None:
            x = self.dropout(x)
            # [N * num_seg, in_channels, 1, 1]
        x = paddle.reshape(x, [-1, num_seg, x.shape[1]])
        # [N, num_seg, in_channels]
        x = paddle.mean(x, axis=1)
        # [N, in_channels]
        x = paddle.reshape(x, shape=[-1, self.in_channels])
        # [N, in_channels]
        score = self.fc(x)
        # [N, num_class]
        #x = F.softmax(x)  #NOTE remove
        return score

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by muare** [comment]
fixxed in #361 . the feature output tensor is added in recognized2.py temporarily before exporting model.


#### step1.3 PP-TSM模型转为预测模式
在转为预测模式前，需要修改 `PaddleVideo/paddlevideo/modeling/framework/recognizers/recognizer2d.py` 文件，将 init 和 infer_step 函数分别更新为如下代码：

```python
def __init__(self, backbone=None, head=None):
        super().__init__(backbone=backbone, head=head)
        self.avgpool2d = paddle.nn.AdaptiveAvgPool2D((1, 1), data_format='NCHW')

def infer_step(self, data_batch):
        """Define how the model is going to test, from input to output."""
        imgs = data_batch[0]
        imgs = paddle.reshape_(imgs, [-1] + list(imgs.shape[2:]))
        feature = self.backbone(imgs)
        feat = self.avgpool2d(feature)
        return feat
```
再执行如下命令：
